ansible_rulebook/cli.py

```diff
@@ -338,6 +344,10 @@ def main(args: List[str] = None) -> int:
     setup_logging_and_display(args)
 
     if args.controller_url:
+        if not validate_controller_url(args.controller_url):
+            print("Error: invalid controller url.")
```

Comment:
```suggestion
            logger.error("Invalid controller url.")
```
We centralize all the output though the logger and I think this should use it.